source/administration/changelog.md

```diff
@@ -4,6 +4,118 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.2
+
+### Highlights
+
+#### Embed Mattermost in Other Apps (Beta)
+ - Added support for extensions, which allow you to embed Mattermost in other apps and websites via OAuth 2.0.
+ - A sample extension for Chrome [is here](https://github.com/mattermost/mattermost-chrome-extension). For documentation on how to write an extension, see here. // XXX Add a link to doc when ready
+
+#### Plugins
+ - Breaking changes to the plugins framework introduced. To migrate your existing plugins to be compatible with Mattermost 5.2 and later, see our [migration guide](https://developers.mattermost.com/extend/plugins/migration/).
+ - Added support to add/delete and enable/disable plugins via the CLI.
+ - See our [demo plugin](https://github.com/mattermost/mattermost-plugin-demo) that demonstrates the capabilities of a Mattermost plugin. For a starting point to write a Mattermost plugin, see our [sample plugin](https://github.com/mattermost/mattermost-plugin-sample).
+ 
+#### Searching Archived Channels
+ - Added ability to search for archived channels.
+ 
+#### Romanian Language
+ - Added support for Romanian language.
+
+### Improvements
+
+#### Web User Interface (UI)
+ - Added experimental custom default channels.
+ - Added link to profile pop-over from names in Join/Leave messages.
+ - Added autolinking to server markdown parser.
```

Comment:
Not sure what this means. Do you have a link to the PR or JIRA tix?

Reply:
@esethna Ticket here: https://mattermost.atlassian.net/browse/MM-11451

Reply:
Not necessary for changelog I don't think